pages/docs/couple-your-code/couple-your-code-time-step-sizes.md

```diff
@@ -86,7 +86,7 @@ while (not simulationDone()){ // time loop
   solverDt = beginTimeStep(); // e.g. compute adaptive dt
   dt = min(preciceDt, solverDt);
   if (precice.isReadDataAvailable()){
-    precice.readBlockVectorData(displID, vertexSize, vertexIDs, displacements);
+    precice.readBlockVectorData(displID, vertexSize, vertexIDs, preciceDt, displacements);
```

Comment:
The surrounding code needs to be updated too. Especially regarding time step size

Reply:
Took care of this via #258. In this PR I mainly wanted to take care of the part with the relative time. Another thing we need to update is the change of `readBlockVectorData` to `readData`. But this is again a todo for another PR or commit (and maybe somebody else :smirk:)

Reply:
dt?
is there still a rebasing missing?

Reply:
Fixed. 3635024f76cd3bd36cddd12e046db2de949cffc4